client/public/editPodcastPage/editPodcastPage.js

```diff
@@ -0,0 +1,27 @@
+window.onload = (event) => {
+  const deleteButton = document.getElementById('delete-btn')
+  deleteButton.parentNode.removeChild(deleteButton)
+}
+const editPodcast = async () => {
+  const editedPodcast = {
```

Comment:
What happens if the user doesn't fill part of the fields and leave them empty? then an empty value will be updated?

Reply:
In the real app we would have the podcast details already as the fields value.
There is an edge case that the admin deletes a field and send it empty (which could be on purpose), I can handle it by having a formatting function that deletes empty fields, the question is if its necessary design wise?

Reply:
I ended up adding a on-liner that filters them.